src/h5web/dimension-mapper/DimensionMapper.module.css

```diff
@@ -43,6 +43,7 @@
 
 .dimSize {
   flex: 1 0 0%;
+  padding: 0 0.1875rem;
```

Comment:
Enforcing minimum distance between dimensions (the space I ended up using is actually two pixels bigger than in the screenshot below):

![image](https://user-images.githubusercontent.com/2936402/107928961-4ab47b00-6f79-11eb-9dfd-09cf7086f8fd.png)